VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts

```diff
@@ -33,7 +33,7 @@ export class UnreadMessagesComponent implements OnInit, OnDestroy {
         );
         this.messagesSubscription$.add(
             this.eventsService.getChatMessage().subscribe(message => {
-                if (message.conferenceId === this.conferenceId) {
+                if (message.conferenceId === this.conferenceId && message.isJudge) {
```

Comment:
This component will not work for participant IMs

Reply:
done